README.markdown

```diff
@@ -210,14 +212,20 @@ Specifies the complete path and location of the MD5 key file containing the keys
 
 Specifies the key identifier to use with the ntpdc utility program. Valid options: value in the range of 1 to 65,534 inclusive. Default value: ' '
 
-#### `keys_trusted`:
+####`keys_template`
```

Comment:
Do you really need this parameter? We don't usually add parameters to change the template used, preferring that PRs be sent to expand the existing templates. So would it work for you to not add this parameter?